Bullwinkle.class.nut

```diff
@@ -95,10 +99,11 @@ class Bullwinkle {
     // Parameters:
     //      name            The message name
     //      data            Optional data
+    //      ts              Optional timestamp for the data
```

Comment:
That is fine (although not what I have implemented currently).  If this is the path forward, I still don't think .ts needs to be sent across the wire or stored twice in RAM... Although that may be a v3 feature